### ISSUE TITLE
I found this projects's preprocess code has a bug

### ISSUE THREAD

**Issue by yangyunfeng-cyber** [TeeyoHuang/Generative-Adversarial-Nets-pytorch#1]
**The code segment about dataloader should change into:**
dataloader = torch.utils.data.DataLoader(
    datasets.MNIST('./data', train=True, download=True,
                   transform=transforms.Compose([
                       transforms.ToTensor(),
                       transforms.Normalize((0.5, ), (0.5,))
                   ])),
    batch_size=args.batch_size, shuffle=True, drop_last=True)
